src/lib/reasoners/intervals.mli

```diff
@@ -19,140 +19,137 @@
 (*                                                                        *)
 (*     CNRS - INRIA - Universite Paris Sud                                *)
 (*                                                                        *)
-(*     Until 2013, some parts of this code were released under            *)
-(*     the Apache Software License version 2.0.                           *)
-(*                                                                        *)
```

Comment:
It's far simpler to use the same header for all the files. So please don't change the header even if no part of the current file existed before 2013. I'll use `headache` to update them before the next release.